js/tab.js

```diff
@@ -46,7 +46,7 @@ define([
   };
   
   //hopefully this never screws up unaugmented Ace sessions.
-  Tab.prototype = EditSession.prototype;
+  Object.assign(Tab.prototype, EditSession.prototype);
```

Comment:
(I have no idea why it doesn't use `Object.create` now, maybe I was particularly low on sleep that week.)